framework/include/base/MooseApp.h

```diff
@@ -642,18 +642,49 @@ class MooseApp : public ConsoleStreamInterface,
    */
   RestartableDataValue & getRestartableMetaData(const std::string & name,
                                                 const RestartableDataMapName & metaname,
-                                                THREAD_ID tid) const;
+                                                THREAD_ID tid);
+
+  /**
+   * Loads the restartable meta data for \p name if it is available with the folder base \p
+   * folder_base
+   */
+  void possiblyLoadRestartableMetaData(const RestartableDataMapName & name,
+                                       const std::filesystem::path & folder_base);
+  /**
+   * Loads all available restartable meta data if it is available with the folder base \p
+   * folder_base
+   */
+  void loadRestartableMetaData(const std::filesystem::path & folder_base);
+
+  /**
+   * Writes the restartable meta data for \p name with a folder base of \p folder_base
+   *
+   * @return The files that were written
+   */
+  std::vector<std::filesystem::path>
+  writeRestartableMetaData(const RestartableDataMapName & name,
+                           const std::filesystem::path & folder_base);
+  /**
+   * Writes all available restartable meta data with a file base of \p file_base
+   *
+   * @return The files that were written
+   */
+  std::vector<std::filesystem::path>
+  writeRestartableMetaData(const std::filesystem::path & folder_base);
 
   /**
    * Return reference to the restartable data object
-   * @return A const reference to the restartable data object
+   * @return A reference to the restartable data object
```

Comment:
So a vector of restartable data maps is "the restartable data object"?

Reply:
Oh yes, a great example of a bad docstring that has gotten even more bad...

I don't even know if these are used anymore. I've gotten rid of almost all direct data access